selection_tools/run_prepare_histograms.sh

```diff
@@ -2,11 +2,11 @@
 
 process=$1
 part=$2
-username=`whoami`
+username=$3
 
 base_input_path="/nfs/dust/cms/user/${username}/ttalps"
-base_output_path="/nfs/dust/cms/user/jniedzie/ttalps"
-# base_output_path="/nfs/dust/cms/user/lrygaard/ttalps" 
+# base_output_path="/nfs/dust/cms/user/jniedzie/ttalps"
```

Comment:
Here we should just insert $username in the path (and assume that the general structure of directories will always be the same).